func x() => return 1; // expect error
